lib/ret_web/controllers/api/v1/media_controller.ex

```diff
@@ -136,4 +113,59 @@ defmodule RetWeb.Api.V1.MediaController do
       :high
     end
   end
+
+  defp maybe_do_telemetry({:commit, nil}), do: Statix.increment("ret.media_resolver.404")
+  defp maybe_do_telemetry({:commit, %Ret.ResolvedMedia{}}), do: Statix.increment("ret.media_resolver.ok")
+  defp maybe_do_telemetry({:error, _reason}), do: Statix.increment("ret.media_resolver.unknown_error")
+  defp maybe_do_telemetry({:commit, :error}), do: Statix.increment("ret.media_resolver.500")
+  defp maybe_do_telemetry({:commit, {:error, _reason}}), do: Statix.increment("ret.media_resolver.500")
+  defp maybe_do_telemetry(_), do: nil
```

Comment:
I only do telemetry on the `{:commit, _}` tuples because that indicates when they are first entered into the cache. This way, we don't log 100 `404`'s if there's 1 resource that 100 people try to reference that only _really_ failed once.